playout/libretime_playout/main.py

```diff
@@ -246,15 +161,11 @@ def run():
 
     liquidsoap_startup_test(telnet_lock, ls_host, ls_port)
 
-    if options.test:
-        g.test_api()
-        sys.exit(0)
-
```

Comment:
This was the only command used from the OptionParser. I removed it as the test_api() function uses non existent function from the api client